app/assets/stylesheets/layout.scss

```diff
@@ -2788,11 +2788,18 @@ table {
     margin-top: rem-calc(-48);
   }
 
+  .cards-container {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
   .card {
     border: 0;
+    flex-grow: 0;
```

Comment:
AFAIK `flex-grow` is `0` by default :thinking:. Is there a reason to explicitely add this line?